pkg/virtctl/guestfs/guestfs.go

```diff
@@ -538,7 +558,7 @@ func CreateAttacher(client *K8sClient, p *corev1.Pod, command string) error {
 }
 
 func (c *guestfsCommand) createInteractivePodWithPVC(client *K8sClient, ns, command string, args []string, isblock bool) error {
-	pod, err := c.createLibguestfsPod(command, args, isblock)
+	pod, err := c.createLibguestfsPod(client, ns, command, args, isblock)
```

Comment:
Since `createLibguestfsPod` now has the client available, you could move the following call to create the pod to `createLibguestfsPod`.